### ISSUE TITLE
Heroku Security Incident / Threat Dragon's Response

### ISSUE THREAD

**Issue by lreading** [OWASP/threat-dragon#419]
**TLDR;**
Threat Dragon was not affected by this incident.  No action is necessary for any Threat Dragon users or maintainers.  This incident does not impact GitHub OAuth Applications, and as such does not impact Threat Dragon users.  Additionally, Threat Dragon does not use any analytics engine or otherwise knowingly collect user data.  Heroku may collect data on application usage, and is subject to their own [Privacy Policy](https://www.heroku.com/policy/security)

**Summary**
Heroku experienced a security incident where Heroku users connected GitHub repositories may have been compromised.  This affected only the "Heroku GitHub" integration, and not the "Heroku Git (CLI)" integration.  Heroku (via Salesforce) has provided the following information on the incident and their response: [https://status.heroku.com/incidents/2413](https://status.heroku.com/incidents/2413) 

**Timeline**
_Apr 15, 2022 11:59pm EDT_ - Heroku sent an email to Threat Dragon maintainers informing of the security incident.  At the time, not many details were available, other than _some_ Heroku customers had their associated GitHub repositories compromised.  We immediately reviewed commit history and repository settings to ensure no changes have been made.  This was communicated between Jon and Leo via private communication via Slack.
_April 16, 2022 8:22am EDT_ - After reviewing the latest status update from Heroku (at the time), we noted that Threat Dragon was not using the affected integration.  Additionally, Leo reviewed his own account's access logs to ensure there were no anomalous activities.
_April 17, 2022 7:26pm EDT_ - Threat Dragon maintainers will be rotating the secrets in the deployed demo instance at [https://www.threatdragon.com/](https://www.threatdragon.com/).  This is not required, but being done out of an abundance of caution and a best practice.  The process for doing this will be added to the documentation for v2.  Additionally, this issue was created to ensure transparency.  
_April 17, 2022 7:31pm EDT_ - Threat Dragon's demo instance's secrets were fully rotated
_April 17, 2022 7:58pm EDT_ - Re-running the latest deploy job from main to ensure that the secret rotation does not impact the demo instance
_April 17, 2022 8:18pm EDT_ - Re-deploy succeeded

**Actions You Need To Take**
N/A

**References**
[Reviewing your GitHub Security Log](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-security-log)
[Heroku Incident 2413](https://status.heroku.com/incidents/2413)

**Threat Dragon's Heroku Configuration**
![Screenshot from 2022-04-17 19-08-57](https://user-images.githubusercontent.com/5851985/163736902-5f31c407-b6fc-403b-9482-9abe5bd1bcb1.png)
![Screenshot from 2022-04-17 19-09-12](https://user-images.githubusercontent.com/5851985/163736906-30a2ad2d-3c65-4db7-8cf8-cd7fb5f2a8b6.png)

This issue will be updated if more information becomes available.

**Comment by jgadsden** [comment]
Thanks for the post on the OWASP slack channel: https://app.slack.com/client/T04T40NHX/CURE8PQ68

I have gone ahead pinned it to the channel
